mrcnn/model.py

```diff
@@ -999,8 +1000,10 @@ def build_fpn_mask_graph(rois, feature_maps, image_meta,
                            name='mrcnn_mask_bn4')(x, training=train_bn)
     x = KL.Activation('relu')(x)
 
-    x = KL.TimeDistributed(KL.Conv2DTranspose(256, (2, 2), strides=2, activation="relu"),
-                           name="mrcnn_mask_deconv")(x)
+    for i in range(num_deconv_layers):
```

Comment:
How about adding a comment explaining that the official paper only has one transposed conv, and that additional layers is an extra feature not described in the paper?

Reply:
comment added right after the explanation of `num_deconv_layers`.